src/layouts/SiteLaunchPad/components/SiteLaunchCheckList.tsx

```diff
@@ -233,9 +248,16 @@ export const SiteLaunchChecklistBody = ({
   let TABLE_MAPPING: TableMappingProps[] = Array.from(
```

Comment:
nit: not an issue with this PR, but `UPPER_SNAKE_CASE` should be reserved for `const`, whicsh this isn't

Reply:
be22fb8ab0e4d0062c20bd8e7ceb2af5a7d1460b